homeassistant/components/evohome/__init__.py

```diff
@@ -187,13 +193,12 @@ def __init__(self, hass, params) -> None:
             access_token = app_storage.get(CONF_ACCESS_TOKEN)
             at_expires_str = app_storage.get(CONF_ACCESS_TOKEN_EXPIRES)
             if at_expires_str:
-                at_expires_dt = as_utc(parse_datetime(at_expires_str))
-                at_expires_dt = at_expires_dt.astimezone(tzlocal())
-                at_expires_dt = at_expires_dt.replace(tzinfo=None)
+                at_expires_local_dt = _utc_to_local_dt(
```

Comment:
We shouldn't ever store anything in local time.

Reply:
Correct. The intention is to store as UTC with no offset.

I'm converting it back to naive local time because the client wants it back.